onnx_tf/backend.py

```diff
@@ -107,6 +110,8 @@ def _onnx_graph_to_tensorflow_rep(cls, graph_def, opset, strict, **kwargs):
     # User provided input tensors, in the case the model inputs have unknown shapes
     input_tensor_dict = kwargs[
         'input_tensor_dict'] if 'input_tensor_dict' in kwargs else dict()
+    training_mode = kwargs[
+        'training_mode'] if 'training_mode' in kwargs else dict()
```

Comment:
Looks like training_mode is a bool flag so the default should be false.

Reply:
Update to False as default value.